lib/auth/password_test.go

```diff
@@ -207,8 +209,11 @@ func (s *PasswordSuite) TestChangePasswordWithOTP(c *C) {
 	c.Assert(err, IsNil)
 
 	otpSecret := base32.StdEncoding.EncodeToString([]byte("def456"))
-	err = s.a.UpsertTOTP(req.User, otpSecret)
-	c.Assert(err, IsNil)
+	dev, err := types.NewTOTPDevice("otp", otpSecret, s.a.clock.Now())
```

Comment:
The clock on the auth server is reset below - does the clock value need to be lifted?

Reply:
done